smalltalksrc/VMMaker/CogVMSimulator.class.st

```diff
@@ -296,7 +296,10 @@ CogVMSimulator >> basicInitialize [
 	traceSources := TraceSources.
 	statCodeCompactionCount := 0.
 	statCodeCompactionUsecs := 0.
-	extSemTabSize := 256
+	extSemTabSize := 256.
+	
+	"This will be overriden when calling initializePluginEntries"
+	mappedPluginEntries := OrderedCollection new.
```

Comment:
This is probably needed too in the StackInterpreterSimulator, no?

Reply:
Indeed, Thanks :) tests are now about 15 sec !